sci-libs/pg2plplot/pg2plplot-5.13.0.ebuild

```diff
@@ -1,9 +1,11 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit cmake-utils fortran-2
+CMAKE_BUILD_TYPE=Release
```

Comment:
Can you see if this can be patched out?
Generally it should be a small 2-3 line patch.
 
The problem is that using `CMAKE_BUILD_TYPE=Release` overrides the `CFLAGS,FFLAGS` from `/etc/portage/make.conf` by using the default cmake flags.

Reply:
I'd like to try, but lack the knowledge.  Can you point me to some documentation?

Reply:
No documentation specifically for this.
Look here - https://sourceforge.net/p/pg2plplot/code/ci/master/tree/CMakeLists.txt#l28
Just remove this check and the `set (CMAKE_CONFIGURATION_TYPES ...` line.